### PR TITLE
[PARIS] JN - Amine

### PR DESCRIPTION
**Pull request by enima2684**
@blksnk

### CODE REVIEW

File: starter_code/views/movies.hbs [hunk 1]
```diff
@@ -1 +1,9 @@
-Hello movies !
\ No newline at end of file
+<div class="movies-page">
+  <h3>Ironhack Cinema</h3>
+  <p>Click on the movie to check the showtime !</p>
+  <ul>
+    {{#each movies}}
```
Comment (abisummers) [inline starter_code/views/movies.hbs]: nice use of a partial!

File: starter_code/views/partials/movieItem.hbs [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<li>
+  <img src="{{ this.image }}" alt="">
```
Comment (abisummers) [inline starter_code/views/partials/movieItem.hbs]: adding alt tags is super important for accessibility

File: starter_code/views/movie.hbs [hunk 3]
```diff
@@ -0,0 +1,25 @@
+<div class="movie-page">
+
+  <div class="left-side">
+    <img src="{{image}}" alt="">
```
Comment (ta-web-paris) [inline starter_code/views/movie.hbs]: adding alt tags is important for accessibility (eg screen readers)

File: starter_code/views/movies.hbs [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
```
Comment (ta-web-paris) [inline starter_code/views/movies.hbs]: nice use of partials!

File: starter_code/app.js [hunk 5]
```diff
@@ -11,7 +11,7 @@ const path         = require('path');
 
 
 mongoose
-  .connect('mongodb://localhost/starter-code', {useNewUrlParser: true})
+  .connect('mongodb://localhost/movie-lab', {useNewUrlParser: true})
```
Comment (ta-web-paris) [inline starter_code/app.js]: I like how you changed the database name rather than just keeping starter-code

**Comment by ta-web-paris** [comment]
Good job! Your site looks exactly like the lab does. Your use of partial is great, it helps keep your code more organised 💪

**Pull request closed without merging**